### PR TITLE
Less ms spam

### PR DESCRIPTION
**Pull request by stonedDiscord**
ok so basically only check the playercount (and reachability) when you hover your mouse over the list

### CODE REVIEW

File: webAO/master.js [hunk 1]
```diff
@@ -64,9 +66,13 @@ export function check_https() {
...
-	console.log(server_description[ID]);
-	if (server_description[ID] !== undefined) {
-		document.getElementById("serverdescription_content").innerHTML = "<b>" + online_counter[ID] + "</b><br>" + server_description[ID];
+	selectedServer = ID;
+
+	if (!lowMemory && document.getElementById(`server${ID}`).className === "")
+		checkOnline(ID, servers[ID].ip + ":" + servers[ID].port);
+
+	if (servers[ID].description !== undefined) {
+		document.getElementById("serverdescription_content").innerHTML = "<b>" + servers[ID].online + "</b><br>" + servers[ID].description;
```
Comment (oldmud0) [inline webAO/master.js:75]: Neither `online` nor `description` are sanitized and can be used for XSS. We need to sanitize them.

**Review by oldmud0 — CHANGES_REQUESTED** [review]

File: webAO/master.js [hunk 2]
```diff
@@ -92,7 +99,15 @@ function onError(evt) {
 }
 
 function checkOnline(serverID, coIP) {
-	let oserv = new WebSocket("ws://" + coIP);
+	let oserv;
+	if (serverID !== -2) {
+		try {
+			oserv = new WebSocket("ws://" + coIP);
+		} catch (SecurityError) {
+			document.getElementById(`server${serverID}`).className = "unavailable";
```
Comment (oldmud0) [inline webAO/master.js:107]:
```suggestion
			document.getElementById(`server${serverID}`).className = "unavailable";
			return;
```

**Review by oldmud0 — CHANGES_REQUESTED** [review]

File: webAO/master.js [hunk 3]
```diff
@@ -173,6 +177,7 @@ function onMessage(e) {
 				+ `<a class="button" href="client.html?mode=join&ip=${ipport}${asset}">Join</a></li>`;			
 		}
 		masterserver.close();
+		return;
```
Comment (oldmud0) [inline webAO/master.js:180]: why did you write this commit

Comment (stonedDiscord) [reply webAO/master.js:180]: return to monke

**Comment by stonedDiscord** [comment]
i resolved it, why does it still say requesting changes

File: webAO/master.js (hunk 3 above)
Comment (stonedDiscord) [reply webAO/master.js:180]: being more serious about it:
i noticed there were more functions that didn't have a return despite being done with their work

**Review by oldmud0 — APPROVED** [review]
idk but here you go

**Pull request merged**